app/src/healthcheck.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+"""
+This is a sample API file that demonstrates how to create an API using FastAPI,
+which is compatible with Chainlit. This file is a starting point for creating
+an API that can be deployed with the Chainlit chatbot.
+"""
+
+import logging
+import os
+import platform
+import socket
+
+from fastapi import APIRouter, Request, status
+from pydantic import BaseModel
+
+healthcheck_router = APIRouter()
+logger = logging.getLogger("chatbot.healthcheck")
```

Comment:
So that all the logs have the same naming format:
```suggestion
logger = logging.getLogger(__name__)
```